foundations/html_css/css-foundations.md

```diff
@@ -23,6 +23,10 @@ At the most basic level, CSS is made up of various rules. These rules are made u
 A `<div>` is one of the basic HTML elements. It is simply an empty container. In general, it is best to use other tags such as `<h1>` or `<p>` for content in your projects, but as we learn more about CSS you'll find that there are many cases where the thing you need is just a container for other elements. Many of our exercises use plain `<div>`s for simplicity. Later lessons will go into much more depth about when it is appropriate to use the various HTML elements.
 </div>
 
+### Values and Units
+
+When you are specifying any property in your CSS file, there could be applicable values depending on the property. To understand what the various values and their units could be, please go through this page [CSS values and units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units).
+
```

Comment:
Because this is such a short amount of content, it feels like this would serve better as an Additional Resource, especially because it is only linking to another resource to read up on the subject.

Reply:
Agree with your perspective. I have moved the link to additional resources section and added one line to direct readers if they wish to learn about values and units.